### ISSUE TITLE
Linux - No compatible binary build found for this package

### ISSUE THREAD

**Issue by HugoHeneault** [serut/meteor-coverage#32]
I got 
```
meteor add lmieulet:meteor-coverage
 => Errors while adding packages:             
                                              
While checking for lmieulet:meteor-coverage@0.9.6:
error: No compatible binary build found for this package. Contact the package author and ask them to publish it for your platform.
```

Why? :sob:

Here's my linux infos:
`Linux pc-linux 4.4.0-36-generic #55-Ubuntu SMP Thu Aug 11 18:01:55 UTC 2016 x86_64 x86_64 x86_64 GNU/Linux
`

**Comment by serut** [comment]
I had a timeout while publishing 0.9.6, so it cannot be used right now. 
Please install the version 0.9.4

**Comment by mafzst** [comment]
@serut You can look at this issue fourseven/meteor-scss#201 He encountered the same issue and he fixed it

**Comment by serut** [comment]
Thanks ! I didn't knew that you could create X.X.X_X, I  try that next time

**Comment by HugoHeneault** [comment]
`meteor add lmieulet:meteor-coverage@=0.9.4` made the thing. :+1: 

Can't wait for the new, shiny, promising, version !

**Comment by serut** [comment]
Indeed, I still don't understand how to publish an atmosphere package 
If you add meteor-coverage right now, you'll get:
```
While checking for lmieulet:meteor-coverage@1.0.0:
error: No compatible binary build found for this package. Contact the package author and ask them to publish it for your platform.
```

I tryed to `publish-for-each` on several build machines (by first using `meteor admin get-machine os.linux.x86_64`), but I always get an error:

```
meteor@ip-172-x-x-x:~$ meteor publish-for-arch lmieulet:meteor-coverage@1.0.0
lmieulet:meteor-coverage: updating npm dependencies -- istanbul-api, body-parser, homedir, minimatch, mkdirp,
remap-istanbul...
=> Errors while initializing project:         
                                              
While building package lmieulet:meteor-coverage:
error: couldn't install npm packages from npm-shrinkwrap: Command failed:
/home/meteor/.meteor/packages/meteor-tool/.1.4.1_1.139xb76++os.linux.x86_64+web.browser+web.cordova/mt-os.linux.x86_64/dev_bundle/bin/npm
install
npm ERR! addLocal Could not install
/tmp/package-source18b58q6/lmieulet_meteor-coverage/.npm/package-new-1iq10df/.cache/minimist/0.0.10/package.tgz
npm ERR! Linux 3.2.0-105-virtual
npm ERR! argv
"/home/meteor/.meteor/packages/meteor-tool/.1.4.1_1.139xb76++os.linux.x86_64+web.browser+web.cordova/mt-os.linux.x86_64/dev_bundle/bin/node"
"/home/meteor/.meteor/packages/meteor-tool/.1.4.1_1.139xb76++os.linux.x86_64+web.browser+web.cordova/mt-os.linux.x86_64/dev_bundle/bin/npm"
"install"
npm ERR! node v4.5.0
npm ERR! npm  v3.10.6
npm ERR! path
/tmp/package-source18b58q6/lmieulet_meteor-coverage/.npm/package-new-1iq10df/.cache/minimist/0.0.10/package.tgz
npm ERR! code ENOENT
npm ERR! errno -2
npm ERR! syscall open
[...]
```

**Comment by serut** [comment]
Fixed with `1.0.1`

```
    "publish": "METEOR_FORCE_PORTABLE=1 meteor publish"
```
